enforcer/datapath.go

```diff
@@ -20,23 +19,6 @@ import (
 	"github.com/aporeto-inc/trireme/policy"
 )
 
-// InterfaceStats for interface
```

Comment:
I dont think we should remove stats. When we start collecting debug stats from enforcerd with the new upcoming feature, we should get these stats to see if we are dropping packets, if we are then look into debug logs. If we forward stats, the enforcer is alive and kicking.